Libraries/Components/View/ViewStylePropTypes.windows.js

```diff
@@ -27,6 +27,7 @@ var ViewStylePropTypes = {
   acrylicOpacityUWP: PropTypes.number,
   acrylicSourceUWP: PropTypes.oneOf(['host', 'app']),
   acrylicTintColorUWP: ColorPropType,
+  reveal: PropTypes.bool,
```

Comment:
>reveal [](start = 2, length = 6)

I have some comments about the name:
  - If you're going to have a UWP suffix on the acrylic props, you should have one on this prop.
  - Does Fluent refer to this feature as "reveal" or "reveal highlight". We should be consistent with Fluent's naming.
  - We should probably include something in the name to indicate it's a boolean (e.g. enabled).

So a better name might be `revealHighLightEnabledUWP`